src/App.js

```diff
@@ -1,11 +1,24 @@
+import React from 'react';
 import './App.css';
+import Dashboard from './Pages/Dashboard';
+import { Route, BrowserRouter as Router, Switch } from 'react-router-dom';
+import PrivateRoute from './components/PrivateRoute';
+import Login from './Pages/Login';
 
-function App() {
```

Comment:
Are we using functions or classes?

Reply:
we agreed on using only functional components

Reply:
juts good to understand Class components, but they are getting outdated, as Hooks are the future :)